content/python/concepts/functions/terms/parameters/parameters.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'Parameters'
```

Comment:
Isn't this term somewhat redundant with `Arguments`? Maybe just have a link back to that entry, and include the new material from this entry there.